_i18n/de/resources/moneropedia/change.md

```diff
@@ -1,15 +1,15 @@
 ---
-terms: ["change"]
-summary: "Monero sent as part of a transaction, that returns to your account instead of going to another recipient"
+terms: ["change", "Wechselgeld"]
+summary: "Monero, die Teil einer Transaktion waren und auf das Konto zurückgebucht werden, anstatt an einen anderen Empfänger zu gehen"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-### The Basics
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+### Grundlagen
 
-Monero sent as part of a transaction, that returns to your account instead of going to another recipient.
+Monero, die als Teil einer Transaktion versendet wurden und auf das Konto zurückgebucht werden, anstatt an einen anderen Empfänger zu gehen.
 
-### More Information
+### Weitere Informationen
 
-The @wallet in the Monero software makes change automatically, but when you send a transaction, you are taking an input that you control and telling the Monero network what to do with it. The input is a "deposit" to your account that you are able to spend. Outputs are the part of the transaction that tells the Monero network where to send the funds.
+Das @Wallet der Monero-Software berechnet Wechselgeld automatisch. Wenn du eine Transaktion vornimmst, sagst du dem Monero-Netzwerk, was es mit einem von dir kontrollierten Input tun soll. Ein Input ist eine "Einzahlung" auf dein Konto, welche du anschließend ausgeben kannst. Outputs hingegen sind Beträge, welche an andere Empfänger gesendet und somit ausgegeben werden.
```

Comment:
Maybe this could be combined? E.g. "Outputs sind der Teil einer Transaktion, welcher dem Monero Netzwerk mitteilt, wohin Beträge gesendet werden sollen (also gewissermaßen die ausgegebenen Geldbeträge)." 
I changed "die dem Monero-Netzwerk mitteilen" to the singular form so that it correctly refers to "der Teil einer Transaktion"; could also be "die Teile" instead of "der Teil" so the following plural form (how you phrased it) would be right.